Cartfile

```diff
@@ -1,3 +1,2 @@
 
-github "Alamofire/Alamofire" ~> 5.4.0
```

Comment:
removing a dependency on Carthage is a breaking change. We will need to tell merchants about this in the release.

Reply:
@harry-brown-cko What is the breaking change ? As i understand since the createCardToken API and others which use the network request now will be replaced by this new code why will this be a breaking change ?

Reply:
Sorry just seen this. The breaking change here is that `Alamofire.xcframework` will no longer be built by Carthage, so if it's referenced as a dependency in Xcode then it won't be found and the build will fail. Merchants will have to modify their project manually to remove Alamofire